Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/BackgroundTask/AddIshBackgroundTask.cs

```diff
@@ -46,13 +50,30 @@ namespace Trisoft.ISHRemote.Cmdlets.BackgroundTask
     /// New-IshSession -WsBaseUrl "https://example.com/ISHWS/" -PSCredential "Admin"
     /// $ishBackgroundTask = Get-IshFolder -FolderPath "General\Myfolder" -FolderTypeFilter @("ISHModule", "ISHMasterDoc", "ISHLibrary") -Recurse |
     ///                      Get-IshFolderContent -VersionFilter Latest -LanguagesFilter en |
-    ///                      Add-IshBackgroundTask -EventType "SMARTTAG"
+    ///                      Add-IshBackgroundTask -EventType "SMARTTAG" -InputDataTemplate IshObjectsWithLngRef
```

Comment:
Remove the explicit added `-InputDataTemplate IshObjectsWithLngRef`.  And enrich the help and this sample description to indicate that the default value of this parameter is `IshObjectsWithLngRef .